### ISSUE TITLE
Renaming of key name

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/python-pytestlog2db#9]
The documentation of PyTestLog2DB contains the following example:

```
"component" {
            "Testsuite1" : "test-data.test_tsclass.TestSuite1",
            "Testsuite2" : "test-data.test_tsclass.TestSuite2",
            "Others" : [
                       "test-data.test_ts1",
                       "test-data.test_ts2"
                       ]
            },
```

But beneath `"component"` more than one component can be defined. Therefore I propose to add a plural 's'.

It's `"components"`, not `"component"`.

**Comment by HolQue** [comment]
Works properly with renamed key.
(retest with latest available feature branches / 28.02.2023)

**Comment by test-fullautomation** [comment]
merged and solved with#20
